docs/pricing/am2-pricing.mdx

```diff
@@ -0,0 +1,262 @@
+
+</Note>
+
+On-demand volume is billed at the end of each billing month. To see on-demand pricing per plan type, check out our [per-category-pricing](#per-category-pricing).
+
+#### Shared and Per-Category On-Demand
+
+Organizations on all plans have the option to set a shared on-demand strategy. Organizations on plans with [performance monitoring](/product/performance/) features have the ability to choose between a shared or per-category on-demand strategy:
+
+- **Shared:** Your on-demand budget is shared between all categories on a first come, first serve basis. There are no restrictions for any single category consuming the entire budget.
+
+- **Per-category:** Your on-demand budgets for categories such as errors, transactions, and replays, are set up individually so that any overages in one category won't consume the budget of another category.
+
+![On-Demand Budgets settings in Subscriptions](./img/on-demand-budgets.png)
+
+### Changing Plans Mid-cycle
+
+If you decide to increase your on-demand budget mid-month, Sentry will start processing additional data as soon as the increase becomes effective.
+
+If you want to decrease your on-demand budget mid-month, but have already consumed more data than the new desired amount allows, your budget will be lowered to match what you’ve already used and any new data will be rejected. Your on-demand budget will be decreased to the desired amount on your next billing date.
+
+If you upgrade from a Team to a Business plan mid-cycle, any existing on-demand budget will be recalculated based on Business tier pricing. This may mean that your on-demand budget will be consumed faster than it was before.
+
+In most cases, new budgets will take effect within minutes. Otherwise, we guarantee a maximum turnaround time of 24 hours.
+
+## Billing Periods
+
+Two different billing periods are available:
+
+- **Billing period** - Your subscription billing period, which is either monthly or annual. This is when we’ll charge the credit card on file for your base subscription and reserved volumes.
+- **On-demand period** - This reflects a one-month window in which we calculate on-demand charges. For monthly subscriptions, on-demand periods are aligned with your billing periods.
+
+Users with the role of [Billing or Owner](/organization/membership/) can find all previous invoices by going to the "Receipts" tab in **Settings > Subscription**.
+
+<Note>
+
+You must have either `Billing` or `Owner` permissions to access or make changes to information on the **Subscription** page. If you don't have those permissions, or you are not a member of the organization in Sentry, please reach out to a Billing member or an Owner to obtain invoices.
+
+</Note>
+
```

Comment:
```suggestion
## Billing Cycles

<Note>
  You have to have Billing or Owner-level permissions to make changes to
  information on the Subscription page or access invoices. If you need an
  invoice, but don't have the right permissions, reach out to your
  administrator.
</Note>

Sentry uses two different billing cycles: one for your paid plan and reserved volume, and another one for your pay-as-you-go usage.

- **Subscription billing cycle:** Billing cycle for your paid plan and reserved volume for which you can elect to be billed on either a monthly or an annual basis.
- **Pay-as-you-go billing cycle:** Billing cycle for your pay-as-you-go usage, for which you'll be billed on a monthly basis. 

_If you're on a monthly subscription billing cycle, your pay-as-you-go billing cycle will be aligned to it._

Users with [Billing or Owner-level](/organization/membership/) permissions can find their invoices by going to the [**Settings**](https://sentry.io/orgredirect/organizations/:orgslug/settings/) page and clicking “Subscription” (under the “Usage & Billing” heading.)

```
Would also update this section as I think we worded the new one more clearly :)

Reply:
```suggestion
## Billing Cycles

<Note>
  You have to have Billing or Owner-level permissions to make changes to
  information on the Subscription page or access invoices. If you need an
  invoice, but don't have the right permissions, reach out to your
  administrator.
</Note>

Sentry uses two different billing cycles: one for your paid plan and reserved volume, and another one for your on-demand volume.

- **Subscription billing cycle:** Billing cycle for your paid plan and reserved volume for which you can elect to be billed on either a monthly or an annual basis.
- **On-demand cycle** - Billing cycle for your on-demand volume. On-demand charges are calculated in a one-month window and billed on a monthly basis.  

_If you're on a monthly subscription billing cycle, your on-demand billing cycle will be aligned to it._
Users with [Billing or Owner-level](/organization/membership/) permissions can find their invoices by going to the [**Settings**](https://sentry.io/orgredirect/organizations/:orgslug/settings/) page and clicking “Subscription” (under the “Usage & Billing” heading.)
```